src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/util/HTTPArgument.java

```diff
@@ -134,14 +134,15 @@ public HTTPArgument(String name, String value, boolean alreadyEncoded) {
     }
 
     /**
-     * Construct a new HTTPArgument instance; alwaysEncoded is set to true.
+     * Construct a new HTTPArgument instance; enabled and alwaysEncoded set to true.
      *
      * @param name the name of the parameter
      * @param value the value of the parameter
      * @param alreadyEncoded true if the name and value is already encoded, in which case they are decoded before storage.
      * @param contentEncoding the encoding used for the parameter value
      */
     public HTTPArgument(String name, String value, boolean alreadyEncoded, String contentEncoding) {
+        setEnabled(true);
```

Comment:
Is this really needed? `enabled` is the default state, so I am not sure what is the reason of adding it here